packages/salesforcedx-vscode-apex/src/commands/forceApexTestRun.ts

```diff
@@ -222,10 +230,33 @@ export class ApexLibraryTestRunExecutor extends LibraryCommandletExecutor<
         payload = { testLevel: TestLevel.RunAllTestsInOrg };
     }
 
-    const result = await testService.runTestAsynchronous(payload, codeCoverage);
+    const progressReporter: Progress<ApexTestProgressValue> = {
```

Comment:
this handles updating the vs code progress based on the events we receive from the `runTestsAsynchronous` operation?

Reply:
Yes